src/esp/assets/ResourceManager.h

```diff
@@ -112,8 +113,25 @@ class ResourceManager {
   static constexpr char PER_VERTEX_OBJECT_ID_MATERIAL_KEY[] =
       "per_vertex_object_id";
 
+  /**
+   * @brief Flag
+   *
+   * @see @ref Flags, @ref flags()
+   */
+  enum class Flag : Magnum::UnsignedShort {
+    /**
+     * build phong material from PBR material
+     */
+    buildPhongFromPbr = 1 << 0,
```

Comment:
```suggestion
    BuildPhongFromPbr = 1 << 0,
```

(also, doesn't Habitat use ALL_UPPERCASE for enums? same would be for the PbrShader flags)

Reply:
OK.

Reply:
> doesn't Habitat use ALL_UPPERCASE for enums?
For example, in the `RenderCamera` class.